### ISSUE TITLE
为什么训练中会出现  Train loss: 3.29611 Val loss:nan...   val loss 怎么运算出了 nan

### ISSUE THREAD

**Issue by dgliu123** [MaybeShewill-CV/bisenetv2-tensorflow#43]
为什么训练中会出现   Train loss: 3.29611 Val loss:nan...  
val loss 怎么运算出了 nan

**Comment by MaybeShewill-CV** [comment]
@dgliu123 大概率还是val数据标签的问题 先检查标签吧

**Comment by dgliu123** [comment]
前面几次val los 是有的 0.20多

**Comment by MaybeShewill-CV** [comment]
@dgliu123 可以降低学习率试试 但我觉得大概率还是标签的问题:)
